packages/block-library/src/html/edit.js

```diff
@@ -15,12 +16,15 @@ import {
 	Disabled,
 	SandBox,
 	ToolbarGroup,
+	PanelBody,
+	ToggleControl,
 } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
 
 export default function HTMLEdit( { attributes, setAttributes, isSelected } ) {
 	const [ isPreview, setIsPreview ] = useState();
```

Comment:
```suggestion
	const [ isPreview, setIsPreview ] = useState( showPreview );
```

Suggestion: We can use attribute value as the initial state. It would be similar to the current effect that only runs on the mount.

Reply:
Hi @Mamaduka thanks for your review, that works perfectly, but we have a new suggestion in the comments below ([here](https://github.com/WordPress/gutenberg/pull/41066#issuecomment-1133769853)) that blocks me to make new commit now.